tests/test_plotting.py

```diff
@@ -826,14 +825,12 @@ def test_plot_b_mag():
     npol = 2
     nzgrid = 128
     alpha = 0
-    # compute and fit iota profile
+    # compute iota
     eq = get("W7-X")
-    data = eq.compute("iota")
-    fi = interp1d(data["rho"], data["iota"])
+    iota = eq.compute("iota", grid=LinearGrid(rho=np.sqrt(psi), NFP=eq.NFP))["iota"][0]
 
     # get flux tube coordinate system
     rho = np.sqrt(psi)
```

Comment:
Could move this up a few lines so you don't have to recompute when computing iota